modules/rtmessageadd.php

```diff
@@ -316,6 +316,8 @@
 			);
 			$headers['X-Priority'] = $RT_MAIL_PRIORITIES[$ticketdata['priority']];
 			$headers['Subject'] = $LMS->ReplaceNotificationSymbols(ConfigHelper::getConfig('phpui.helpdesk_notification_mail_subject'), $params);
+			if(ConfigHelper::checkConfig('rt.note_send_re_in_subject'))
```

Comment:
this setting applied only to notes (as its name suggests). Do you want to extend its range to messages to?